webui/react/src/components/Page.tsx

```diff
@@ -22,6 +25,7 @@ interface Props extends CommonProps {
   subTitle?: React.ReactNode;
   title?: string;
   showDivider?: boolean;
+  seoTitle?: string;
```

Comment:
question: SEO? this isn't just for getting indexed by google is it?
would document title be more accruate

Reply:
to understand the prop better could you add comments on what do you think this would get set to compared to title

Reply:
I used `seoTitle` because there's already a `title` there. Open to any other name, seo is because we're taking of the <head><title>, and should make it obvious. But we can call it `browserTitle`, `headTitle`, `tabTitle`, `pageTitle`, anything you think is best just lmk @hamidzr.

And yes I can add comments.

Reply:
nit: hmm, I was thinking SEO might be a bit confusing as the title is updated at run time with javascript so some SE bots won't catch it. all of the other ones you suggested sounds good to me maybe other than `pageTitle` since it's the page component and it already has a title prop as well.

thanks!

Reply:
**nit suggestion:** how about `docTitle`?